db/seeds.rb

```diff
@@ -7,8 +7,18 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 User.create(username: "test", email: "test@m.com", password: "12345678")
 
-energy = Title.create(name: "Energy use (kg of oil equivalent per capita)")
-biodiversity = Title.create(name: "GEF benefits index for biodiversity")
+Title.create([
+  { value: "AG.LND.FRST.ZS", name: "Forest area (% of land area)" },
+  { value: "EG.USE.PCAP.KG.OE", name: "Energy use (kg of oil equivalent per capita)" },
+  { value: "EN.POP.DNST", name: "Population density (people per sq. km of land area)" },
+  { value: "EG.FEC.RNEW.ZS", name: "Renewable energy consumption (% of total final energy consumption)" },
+  { value: "AG.LND.FRST.K2", name: "Forest area (sq. km)" },
+  { value: "AG.LND.IRIG.AG.ZS", name: "Agricultural irrigated land (% of total agricultural land)" },
+  { value: "ER.LND.PTLD.ZS", name: "Terrestrial protected areas (% of total land area)" },
+  { value: "BN.GSR.FCTY.CD", name: "Net primary income (BoP, current US$)" },
+  { value: "EG.GDP.PUSE.KO.PP", name: "GDP per unit of energy use (PPP $ per kg of oil equivalent)" },
+  { value: "EG.GDP.PUSE.KO.PP.KD", name: "GDP per unit of energy use (constant 2011 PPP $ per kg of oil equivalent)" }
+])
```

Comment:
Indent the right bracket the same as the first position after the preceding left parenthesis.